cmd/core-data/main.go

```diff
@@ -17,22 +17,20 @@ package main
 import (
 	"flag"
 	"fmt"
-	"net/http"
 	"os"
 	"os/signal"
 	"strconv"
 	"time"
 
-	"github.com/edgexfoundry/edgex-go"
 	"github.com/edgexfoundry/edgex-go/internal"
 	"github.com/edgexfoundry/edgex-go/internal/core/data"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/correlation"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/startup"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
+
+	"github.com/edgexfoundry/edgex-go"
```

Comment:
Should come before `github.com/edgexfoundry/edgex-go/internal`. Please review all as it looks like this is all of the main.go files

Reply:
Is this considered being part of the internal package? I'll move it, I just didn't know.